apps/pxweb2/src/app/components/Selection/Selection.module.scss

```diff
@@ -0,0 +1,11 @@
+@use '$ui/style-dictionary/dist/scss/fixed-variables.scss' as fixed;
+.variableListContainer {
```

Comment:
I think this one is removed. Styling is done directly on VariableList i think

Reply:
Style removed